client/driver/lxc.go

```diff
@@ -168,6 +168,10 @@ func (d *LxcDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool, e
 	return true, nil
 }
 
+func (d *LxcDriver) Prestart(ctx *ExecContext, task *structs.Task) error {
+	return nil
```

Comment:
Not sure I follow. TaskEnv is built in TaskRunner.Run before the driver is created and passed into the NewDriver factory func via DriverContext, so it's already built and available at this point.